web-module/src/main/webapp/resources/celJS/images/imageSlideShow.js

```diff
@@ -941,10 +945,128 @@ window.CELEMENTS.image.SlideShow = function(config) {
       _startStopClickHandler : function(event) {
         var _me = this;
         event.stop();
-        _me.startStop();
+        var clickedElement = event.findElement();
+        var linkHref = '';
+        if(clickedElement.up('div.cel_slideShow_slideWrapper a')) {
+          linkHref = clickedElement.up('div.cel_slideShow_slideWrapper a').href;
+        }
+        $(document.body).observe('click', _me._contextMenuSlideShowListItemClickedBind);
+        if(linkHref && (linkHref !== '')) {
+          var slideShowWrapper = clickedElement.up('.celimage_slideshow_wrapper');
+          var rect = slideShowWrapper.getBoundingClientRect();
+          var mouseCoord = _me._getMousePos(event);
+          var x = mouseCoord[0] - 6 - rect.left;
+          var y = mouseCoord[1] - 3 - rect.height;
+          _me._menuDiv = slideShowWrapper.down('.contextMenuSlideShow');
+          if(_me._menuDiv == null) {
+            _me._menuDiv = _me._generateMenuDiv(clickedElement);
+            slideShowWrapper.insert(_me._menuDiv);
+          } else {
+            _me._menuDiv.show();
+          }
+          $$('.contextMenuSlideShowListItem').each(function(element) {
+            element.observe('click', _me._contextMenuSlideShowListItemClickedBind)
+          });
+          _me._setPosition(x, y);
+          _me.startStop(false, undefined, true);
+        } else if(clickedElement.nodeName.toLowerCase() !== 'img') {
```

Comment:
why check for "!== 'img'"? A Slide is just a snippet of html. It could be a table, paragraph, list or everything combined. A slide does not necessarily exist of only one img-tag.